deploy.py

```diff
@@ -91,15 +105,15 @@ def push(ctx, target, version=None):
     ctx.run(f"docker push {repository}/{name}:{version}", echo=True, pty=True)
```

Comment:
Hoef je nu niks mee te doen, maar zouden we die task definitions via terraform kunnen provisionen als we bepaalde versies gaan taggen bij een deploy? Je ziet natuurlijk vaak `:latest` voor de laatste versie. Kunnen we bij een deploy dan niet een versie taggen met `:development` of `:acceptance`? Dan hoeven we niet de task definition te updaten, maar alleen de service opnieuw te starten.

Zie: https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-retag.html

En dan vervolgens: `aws ecs update-service --cluster <cluster name> --service <service name> --force-new-deployment`

Reply:
Ja, "latest" is een anti-pattern, want het doet vaak niet wat je wilt. Zie: https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375

Ik ben fan van prd, acc en dev tags (doe ik voor mijn eigen projecten ook). Dat is goed te doen, maar je moet dan tags pushen naar AWS en je moet ook tags pullen om te weten welke versie beschikbaar is. Dat gaat niet via de standaard Docker interface, maar via API's. Ik heb voor GCloud scripts liggen die het doen.

Dus ja, ik ben opzich voor, maar inderdaad out-of-scope voor ticket.